CHANGELOG.md

```diff
@@ -10,3 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - New `NGINX_HIDE_DRUPAL_HEADERS` environment variable to hide the drupal information from the response headers (default: the headers are visible)
 - New `NGINX_HIDE_SENSITIVE_HEADERS` environment variable to hide all the sensitive information from the response headers (default: the headers will be removed)
+
+## 2022-02-03
+
+### Added
+
+- New `NGINX_OSB_HIDE_GOOGLE_HEADERS` environment variable to hide the google response headers coming from the google object storage bucket (default: the headers are hidden)
```

Comment:
Just a bout the naming convention, I would prefer to be more specific here:

HIDE_GOOGLE_GCS_HEADERS

just to better understand what headers of what service we are hiding